example/App.js

```diff
@@ -224,6 +224,16 @@ export const App = () => {
                 title="hide picker"
               />
             </View>
+            <View style={styles.button}>
+              <Button
+                testID="setTzZero"
+                onPress={() => {
+                  setTzOffsetInMinutes(0);
+                  setShow(true);
+                }}
+                title="setTzZero"
```

Comment:
can you please give this a more explanatory title? There is enough space in the button 🙂

Reply:
hi, this does not seem to be resolved, can you take another look? It's about the button titles that are visible to users (`setTzZero` and `setTz`). Thank you! :)

Reply:
This is resolved. I will add more to title